### PR TITLE
Expense list updated: Expense list updated with state and map

### PR DESCRIPTION
**Pull request by khushboogoyal88**

### CODE REVIEW

**Comment by khushboogoyal88** [comment]
I am however now confused, shud I have instead put the state in Expense page  or directly in App.js

File: frontend/src/components/ExpenseList.js [hunk 1]
```diff
@@ -4,17 +4,36 @@ import { FaSave } from "react-icons/fa";
 import "../styles/ExpenseList.css";
 
 class ExpenseList extends React.Component {
+  state = {
+    expenses: [
```
Comment (vesasuikkola) [inline frontend/src/components/ExpenseList.js]: Please rename to "categories"

File: frontend/src/components/ExpenseList.js [hunk 2]
```diff
@@ -4,17 +4,36 @@ import { FaSave } from "react-icons/fa";
...
+        categoryid: 8,
+        categoryName: "Utility bills (cable, cell, electricity, water, etc",
+      },
+      { categoryid: 9, categoryName: "Health expenses" },
+      { categoryid: 10, categoryName: "Services (daycare, hair dresser)" },
+      { categoryid: 11, categoryName: "Miscellenous" },
+    ],
+  };
   render() {
+    const expenseList = this.state.expenses.map((item) => {
```
Comment (vesasuikkola) [inline frontend/src/components/ExpenseList.js]: Please rename to categoryList

**Review by vesasuikkola — CHANGES_REQUESTED** [review]

**Comment by vesasuikkola** [comment]
> I am however now confused, shud I have instead put the state in Expense page or directly in App.js

This is ok for now, we'll anyways need to move the list to localstorage/backend later. I'll raise a separate PBI for that.

**Comment by khushboogoyal88** [comment]
I have made the name convention changes as you requested. Please have a look.

**Review by vesasuikkola — DISMISSED** [review]

File: frontend/src/components/ExpenseList.js (hunk 1 above)
Comment (vesasuikkola) [reply frontend/src/components/ExpenseList.js]: 12c05b0ab8d6a0ff42c54ec8615cd663695e66a6 OK

File: frontend/src/components/ExpenseList.js (hunk 2 above)
Comment (vesasuikkola) [reply frontend/src/components/ExpenseList.js]: 12c05b0ab8d6a0ff42c54ec8615cd663695e66a6 OK

**Review by SaharPerveen — DISMISSED** [review]
ok

**Review by vesasuikkola — APPROVED** [review]

**Review by VilmaIkonen — APPROVED** [review]

**Pull request merged**